README.md

```diff
@@ -0,0 +1,49 @@
+# Afkanerd-OpenWebSockets
+
+## **Setup and Config**
+
+### **Global config**
+
+- Clone or download repository to local machine.
+
+  > `git clone https://github.com/Wisdom-Nji/Afkanerd-OpenWebSockets.git`
+
+- Install dependencies with npm.
+
+  > `npm install`
+
+- Grant execute permission to install script.
+
+  > `sudo chmod 777 install.sh`
+
+- Run install script to configure client.
+
+  > `sudo ./install.sh`
+
+### **Server Setup**
+
+- Switch to server folder.
+
+  > `cd server`
+
+- Start server script with pm2.
+
+  > `pm2 start main.js -n "name your server instance"`
```

Comment:
You don't need pm2 if you're not running as a daemon, e.g when developing.
But before pm2 install pm2
`npm install -g pm2`